src/infrastructure/middlewares/user.middleware.js

```diff
@@ -0,0 +1,16 @@
+const userService = require('resources/user/user.service');
+
+const userMiddleware = async (ctx, next) => {
```

Comment:
I think this middleware also could reside near routes as it used only there, pretty much in one place.